services/graphql/src/definitions/customer.js

```diff
@@ -7,8 +7,10 @@ extend type Query {
   changedCustomers(input: ChangedCustomersQueryInput!): [ChangedCustomer!]!
   "Finds a single customer by customer ID."
   customerById(input: CustomerByIdQueryInput!): Customer
+  "Deprecated: use customerByEncryptedId instead."
+  customerByEncyptedId(input: CustomerByEncryptedIdQueryInput!): Customer @deprecated(reason: "Use customerByEncryptedId instead.")
   "Finds a single customer by encrypted customer ID."
-  customerByEncyptedId(input: CustomerByEncryptedIdQueryInput!): Customer
+  customerByEncryptedId(input: CustomerByEncryptedIdQueryInput!): Customer
```

Comment:
Is the typo query being used in the wild anywhere?

Reply:
> Is the typo query being used in the wild anywhere?

In FII, yes

Reply:
https://github.com/parameter1/form-identity-integrations/blob/main/api/src/omeda/load-customer.js#L15